app/Models/AssetMaintenance.php

```diff
@@ -95,8 +95,8 @@ public function setIsWarrantyAttribute($value)
      */
     public function setCostAttribute($value)
     {
-        $value = Helper::ParseFloat($value);
-        if ($value == '0.0') {
+        $value = Helper::ParseCurrency($value);
+        if ($value == 0) {
```

Comment:
Hi @dboth - did you have a chance to look at @uberbrady's question here?

Reply:
I'm not sure about this. It depends if you want to be able to distinguish between a purchase cost of $0 and no purchasing cost. For my part and for the use cases in our company I don't see the added value in making a difference here and I don't see what benefit it would bring. 

However, if we want to distinguish between 0 and null, this is not the only place to change it. The `Helper::ParseCurrency` method always returns a float, so the user entering `""` into the input field has the same effect as them entering `"0"`.